Update README.md

lisaychuang opened the pull request:
Added 3 links featued in ElaConf newsletter:
* What You Need to Know About Speaking at Conferences - Ashe Dryden
* My presentation workflow and challenges - Lara Hogan
* Finding Your Killer Talk Idea - Rachel Nabors

lisaychuang committed:
Update README.md

Added 3 links featued in ElaConf newsletter:
* What You Need to Know About Speaking at Conferences - Ashe Dryden
* My presentation workflow and challenges - Lara Hogan
* Finding Your Killer Talk Idea - Rachel Nabors

lisaychuang committed:
Update README.md

Added a link by by Damir Svrtan.

lisaychuang commented:
Added a link by by Damir Svrtan.

vmbrasseur commented on README.md:
```diff
@@ -70,6 +72,7 @@ Participation" depending on the conference.
 
 ### Proposing talks
 
+* [Finding Your Killer Talk Idea](http://ladiesintech.com/finding-your-killer-talk-idea/) by Rachel Nabors
```
Could you please add this to the end of this section's list rather than the start?

(I should add it to the CONTRIBUTING guide; I'll do this momentarily).

lisaychuang committed:
Update README.md

Moved Rachel Nabors link to the end of PROPOSING TALKS section:
[Finding Your Killer Talk Idea](http://ladiesintech.com/finding-your-killer-talk-idea/) by Rachel Nabors

lisaychuang commented:
Done! Moved Rachel Nabors' link to end of PROPOSING TALK section.

vmbrasseur commented:
Woo! Thank you, @lisaychuang! ❤️
